src/web/lib/components/AppHeader/index.js

```diff
@@ -43,7 +43,7 @@ export const AppHeader = props => {
     if (shouldUpdate) {
       syncImages();
     }
-  }, [props.themeCustomBackgrounds]);
+  }, [shouldUpdate]);
```

Comment:
This change is correct iff the module only needs to account for changes initiated through the UI, not for external changes (such as changes in storage, e.g. by changing themes in a different tab).

Can you confirm whether this is your intended change?

Reply:
hmm.. it seems that either way, adding a custom image in another tab will cause a similar issue (#877).  In another issue #849, we decided to clear out the image from local storage if the user refreshed and the image wasn't saved so not too sure how to get around this. I can check for the image before displaying it in the background list but regardless it seems that it's irrelevant to this change here.

Reply:
@Rob--W, I am wondering if we can merge this in as is (or update the default.json file). Either way seems to fix the issue at hand within the same tab but if you open in another tab, it's still a problem. To me this seems less common UI (?) and since it currently does this - so it's not something that this PR is breaking - I am wondering if we can have a follow up issue to look into another solution (keeping in mind #849). 

What do you think?